packages/xstate-wallet/package.json

```diff
@@ -13,7 +13,7 @@
     "@statechannels/client-api-schema": "0.0.1",
     "@statechannels/nitro-protocol": "0.0.1",
     "@statechannels/wallet-protocols": "1.0.0",
-    "ethers": "4.0.40",
+    "ethers": "4.0.39",
```

Comment:
Why downgrade?

Reply:
To keep it consistent with `nitro-protocol` and `wallet`.  I was getting issues with `BigNumber` from `ethers` not matching between `nitro-protocol` and `xstate-wallet`.